Sources/SentryCrash/Recording/SentryCrash.m

```diff
@@ -67,10 +67,19 @@
 static NSString *
 getBundleName(void)
 {
-    NSString *bundleName = [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleName"];
+    static NSString *bundleName = nil;
+
     if (bundleName == nil) {
-        bundleName = @"Unknown";
+#if TEST
+        bundleName = @"Sentry/Test";
```

Comment:
I think this is very dangerous. In many componentized projects, developers bring some macro definitions to pod components, and TEST is a very common one, used as a marker in test environments. In this case the bundleName will always be "Sentry/Test"

Reply:
Although I believe a dev should not add a macro definition to a external library pod this is a fair point, but we do have other places with TEST in the project.
We probably should rethink this.

Reply:
> Although I believe a dev should not add a macro definition to a external library pod this is a fair point, but we do have other places with TEST in the project.
> We probably should rethink this.

In a componentized project, although we can use some hard coding to distinguish which are business components and which are third-party components, some inexperienced developers may ignore this and bring macro definitions to all pods.

I think the right approach is that the third-party pod and the main app each ensure the uniqueness of their own macro definitions.